learn/advanced/known_limitations.md

```diff
@@ -115,12 +115,12 @@ user = 1 OR user = 2 […] OR user = 1500 OR user = 1501 […] OR user = 2000 OR
 
 ## Maximum database size
 
-**Limitation:** The maximum size of an index is 500GiB, and the maximum size of the task database is 10GiB.
+**Limitation:** The maximum index size Meilisearch supports on Linux environments is around 80TiB. For performance reasons, Meilisearch recommends keeping indexes under 2TiB.
 
-**Explanation:** Meilisearch allocates all the virtual memory it requires upfront. The maximum database size ensures instances can contain several large indexes without reaching operating system limits on the amount of virtual memory available to a single process.
+**Explanation:** Meilisearch can accommodate indexes of any size as long the combined size of active databases is below the maximum virtual address space the OS devotes to a single process. On Linux, this limit is approximmately 80TiB.
```

Comment:
```suggestion
**Explanation:** Meilisearch can accommodate indexes of any size as long the combined size of active databases is below the maximum virtual address space the OS devotes to a single process. On Linux, this limit is approximately 80TiB.
```

Reply:
Dishonour on my family! Thank you for catching that typo 👍